README.md

```diff
@@ -106,22 +106,23 @@ When building C/C++ software, often the build is performed in parallel via a com
 If you are in an Ubuntu docker container, omit `sudo` from all commands because you run as `root` by default. Most other docker containers also exclude `sudo`, especially Debian-family containers. If your shell prompt is a hash tag (`#`), omit `sudo`.
 
 #### Pinned Build
-Make sure you are in the root of the `leap` repo, then run the `install_depts.sh` script to install dependencies:
-```bash
-sudo scripts/install_deps.sh
-```
-
-Next, run the pinned build script. You have to give it three arguments in the following order:
+Make sure you are in the root of the `leap` repo, then run the pinned build script. You have to give it three arguments in the following order:
 1. A temporary folder, for all dependencies that need to be built from source.
 1. A build folder, where the binaries you need to install will be built to.
 1. The number of jobs or CPU cores/threads to use (note the [jobs flag](#step-3---build) warning above).
 
-> 🔒 You do not need to run this script with `sudo` or as root.
+> 🔒 You do not need to run this script with `sudo` or as root. On Debian-family operating systems such as Ubuntu, the script will determine if all package dependencies are installed and only invoke aptitude if you are missing packages. In that case, you may be prompted for your password if aptitude needs additional permissions from `sudo`. You should not run this script with `sudo` or as root because you do not need to.
```

Comment:
It looks as though we call `apt-get` in the script, not `aptitude` or `apt` alone.

Reply:
Yes, `apt-get` is for scripts, `apt` is for humans. Changed.